internal/config/env.go

```diff
@@ -61,9 +61,11 @@ func LoadEnv() error {
 		env = "local"
 	}
 
-	err := godotenv.Load(fmt.Sprintf(".env.%s", env))
-	if err != nil {
-		return err
+	if env != "develop" {
```

Comment:
instead of doing this can, lets check if the file exists and if it exists we load it, else we don't